Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -236,6 +244,7 @@
   - type: Clothing
     sprite: Objects/Weapons/Guns/Battery/laser_gun.rsi
   - type: Gun
+  - fireRate: 2 # DeltaV - parent changed to 1.5
```

Remove the - in those, YAML doesn't work like that. I believe that's what is causing all the test fails.
```suggestion
    fireRate: 2 # DeltaV - parent changed to 1.5
```